plugins/covid/plugin.py

```diff
@@ -0,0 +1,33 @@
+"""
+Plugin definition for the Covid Opal plugin
+"""
+from opal.core import plugins, menus
+from opal.models import UserProfile
+
+from plugins.covid.constants import COVID_ROLE
+from plugins.covid.urls import urlpatterns
+
+
+class CovidPlugin(plugins.OpalPlugin):
+    urls = urlpatterns
+
+    @classmethod
+    def get_menu_items(self, user):
+        if not user:
+            return []
+
+        dashboard = menus.MenuItem(
+            href='/#/covid-19/',
+            display='Covid 19',
+            icon='fa fa-dashboard',
+            activepattern='/#/covid-19/'
+        )
+
+        profile = UserProfile.objects.get(user=user)
+        if profile.roles.filter(name=COVID_ROLE).exists():
```

Comment:
if I'm `user.profile.roles.filter(name=COVID_ROLE).exist()` looks a bit nicer but I that's just my aesthetic opinion (very unimportant)